### PR TITLE
Adding a terraform on GCP tutorial

### PR DESCRIPTION
**Pull request by chrisst**
This tutorial will show somebody how to start up a flask app serving traffic on a google compute engine instance using terraform to provision. It is meant for somebody who is unfamiliar with either tool and will allow them to get a very basic introduction.

### CODE REVIEW

**Comment by chrisst** [comment]
@ryanmcginnis that's fine. It does look like many of the new tutorials require multiple reviewers though, so in the mean time would you be able to add labels and add the other reviewers?

File: tutorials/getting-started-on-gcp-with-terraform/index.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+ allow {
+   protocol = "tcp"
+   ports    = ["5000"]
+ }
+}
+```
+Congratulations, you can now point your browser to the instance’s ip address / port 5000 and see your server running!
+
+## Cleaning up
+Now that you are finished with the tutorial you will likely want to tear down everything that was created so that you don’t incur any further costs. Thankfully Terraform will let you remove all the resources defined in the configuration file with `terraform destroy`:
```
Comment (dghale) [inline tutorials/getting-started-on-gcp-with-terraform/index.md]: :s/Thankfully/Thankfully,

**Comment by chrisst** [comment]
Thanks @dghale, edits made!

**Comment by jmdobry** [comment]
Published to https://cloud.google.com/community/tutorials/getting-started-on-gcp-with-terraform

Please review and open another PR with any changes you need to make.

**Pull request merged**